doc/TBG_macros.cfg

```diff
@@ -188,6 +188,30 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# Live in situ visualization using ISAAC
+# --isaac.period PERIOD
+#   ↳ Initial period in which a image shall be rendered
```

Comment:
You destroyed a masterpiece! https://github.com/ComputationalRadiationPhysics/picongpu/pull/1630/commits/bc76aece10ae170e2ba4010fe251952cd84c71d4